src/esm_tests/tests.py

```diff
@@ -789,9 +789,9 @@ def exist_files(files, path, version):
         exception_list = find_exceptions(f)
 
         # Command's logic
-        if " except " in f and version in exception_list:
+        if " except " in f and str(version) in exception_list:
```

Comment:
This makes me cry on the inside. How about:

```suggestion
        version_in_exception_list = str(version) in exception_list
        version_not_in_exception_list = not version_in_exception_list
        f_contains_except = " except " in f
        f_contains_in = " in " in f
        if f_contains_except and version_in_exception_list:
            continue
        elif f_contains_in and version_not_in_exception_list:
            continue
            
```